virtool/labels/db.py

```diff
@@ -0,0 +1,2 @@
+async def attach_sample_count(db, document, label_id):
+    document.update({"count": await db.samples.count_documents({"labels": {"$in": [label_id]}})})
```

Comment:
This should work:
```py
await db.samples.count_documents({"labels": label_id})
```

Right now this function relies on a side-effect to work: it modifies `document` from the outside scope. A purer approach is to return a new `dict`:

```py
async def attach_sample_count(db, document, label_id):
  return {
    **document,
    "count": await db.samples.count_documents({"labels": {"$in": [label_id]}})
  }
```